files/image_config/caclmgrd/caclmgrd

```diff
@@ -247,6 +304,63 @@ class ControlPlaneAclManager(object):
 
                             iptables_cmds.append(rule_cmd)
 
+        # For any services which we did not find ACL configuration for, we add an
+        # iptables rule to allow all incoming packets. This way we don't block
+        # services like SSH on an unconfigured device
+        for acl_service in unconfigured_services:
+            log_info("Did not find or could not apply ACL configuration for {0}. Allowing all incoming traffic for this service...".format(acl_service))
+            ip_protocols = self.ACL_SERVICES[acl_service]["ip_protocols"]
+            dst_ports = self.ACL_SERVICES[acl_service]["dst_ports"]
+            for ip_protocol in ip_protocols:
+                for dst_port in dst_ports:
+                    iptables_cmds.append("iptables -A INPUT -p {0} --dport {1} -j ACCEPT".format(ip_protocol, dst_port))
+                    iptables_cmds.append("ip6tables -A INPUT -p {0} --dport {1} -j ACCEPT".format(ip_protocol, dst_port))
+
+        # Add iptables rules to drop all packets destined for loopback interface IP addresses
+        # (and in the case of V6, all point-to-point addresses in the /64 subnet)
+        loopback_iface_table = self.config_db.get_table(LOOPBACK_INTERFACE_TABLE)
+        for ((iface_name, iface_cidr), _) in loopback_iface_table.iteritems():
+            ip_ntwrk = ipaddress.ip_network(iface_cidr, strict=False)
+            if isinstance(ip_ntwrk, ipaddress.IPv4Network):
+                iptables_cmds.append("iptables -A INPUT -d {}/32 -j DROP".format(ip_ntwrk.network_address))
+            elif isinstance(ip_ntwrk, ipaddress.IPv6Network):
+                iptables_cmds.append("ip6tables -A INPUT -d {}/64 -j DROP".format(ip_ntwrk.network_address))
```

Comment:
change this to /128.

can you add a series of new rules to block all traffic to p2p address, including both v4 and v6. then, we will use a unified way to handle both v4 and v6 p2p addresses.

Reply:
In latest commit, I am explicitly adding drop rules for all P2P addresses. Please review.